src/disk91_LoRaE5.cpp

```diff
@@ -636,6 +653,122 @@ bool Disk91_LoRaE5::setup(  // Setup the LoRaWAN stack
     return true;
 }
 
+bool Disk91_LoRaE5::setup_sensecap(  // Setup the LoRaWAN stack
```

Comment:
I'm a bit lost with what this method is doing compared to the other method. 
Need some comments to explain what is different with a standard setup, what for it is used.